valimail.com.valimail-authenticate.json

```diff
@@ -0,0 +1,38 @@
+{
+  "providerId": "valimail.com",
+  "providerName": "Valimail",
+  "serviceId": "valimail-authenticate",
+  "serviceName": "Valimail Authenticate",
+  "version": 1,
+  "logoUrl": "https://valimail-logos.s3.amazonaws.com/valimail-logo-primary.svg",
+  "description": "Enable Valimail Authenticate on a domain",
+  "variableDescription": "%spftxt% is the SPF record",
+  "records": [
+      {
+      "type": "NS",
+      "host": "_dmarc",
```

Comment:
Generally we discourage non-conditional usage of DMARC, as it applies to the whole domain, not only to one service.

Reply:
Hi Pawel, thanks for the feedback. I think there's a general misunderstanding here, and I'd like to attempt to clarify what we do, and exactly what this template is asking for:

Our customers do NS delegations of _dmarc, _domainkey, and _bimi, and create a special SPF TXT record, and then our app provides management and maintenance of everything at or under those labels. We're able to auto-import the existing data through other means, so these actions are not destructive.

So yes, we fully agree that DMARC is about the entire domain. That's why the above delegations are what we ask for-- we're then able to manage and authenticate each individual service without our customers ever needing to make manual DNS changes again.

We do this all the time, for instance, check out NS _dmarc.uber.com or _domainkey.epa.gov or _bimi.hbo.com (or the v=spf1 TXT record on each of those domains).

With domain connect, our customers won't even need to make 3 NS delegations and 1 TXT record update, they'll just need to press the button they've been asking for.

Let me know if that makes sense, we're happy to clarify.

Reply:
Thanks, this explanation was quite helpful.
Looks like your service is similar to [FraudMarc](https://github.com/Domain-Connect/Templates/blob/master/fraudmarc.com.mail.json).

Then I don't have any further questions actually and I can approve this template.
@arnoldblinn would you take a look as well?